parser/regexp.txt

```diff
@@ -0,0 +1,112 @@
+// Used in SIMILAR etc. For now, we call it unsupported.
+void regular_expression() #Unsupported:
```

Comment:
Is the regexp from spec? I think we allow different regexp libraries in presto?

Reply:
We simply use a string literal for regex. But the spec actually has a full regex spec in the grammar! We have that fragment of the grammar in the repo as well if we want to eventually support that.

Reply:
Hmm what does "unsupported" here mean? CoreSQL not supporting or Presto? There are probably more stuff that Presto doesn't support?

Reply:
> Hmm what does "unsupported" here mean? CoreSQL not supporting or Presto? There are probably more stuff that Presto doesn't support?

I talk about it in the notes now. Basically, any syntax that we don't support right now turns into an 'Unsupported' AST node so that we can give good error messages in the right places.